EIPS/eip-8094.md

```diff
@@ -55,7 +55,7 @@ This is the response to GetPooledBlobs, returning the requested blobs. The items
 >
 > It is important to include all cell proofs to keep reconstruction CPU-efficient.
 
-The blobs must be in the same order as in the request, but it is OK to skip blobs which are not available. Since the recipient have to check that transmitted blob hashes correspond to the requested vhashes anyway, we can avoid sending the list of vhashes as part of this message.
+The blobs must be in the same order as in the request, but it is OK to skip blobs which are not available. Since the recipient have to check that transmitted blob hashes correspond to the requested vhashes anyway, we can avoid sending the list of vhashes as part of this message. If some blobs are unavailable, the recipient may request them from other peers or wait for them to become available before considering the transaction complete.
```

Comment:
Yes, it can request from other peers. What do you mean by "become available"?

I was keeping it OK to skip blobs to mirror what we have for transactions currently in the spec. However, it is expected that nodes keep announced transaction and blobs around for a while, so such "skips" should not be frequent. If they are frequent, it is reasonable to disconnect such peer (this is not something I would specify, but leave to implementation).